pkg/bpf/bpf.go

```diff
@@ -140,6 +149,7 @@ func (l *BpfLoader) Stop() {
 	var err error
```

Comment:
There is no need to define an error first

Reply:
ok have updated PR desc